src/ducks/balance/helpers.js

```diff
@@ -29,8 +29,14 @@ export const getBalanceHistories = (accounts, transactions) => {
   return balances
 }
 
-const getTransactionsForAccount = (account, transactions) =>
-  transactions.filter(t => t.account === account._id)
+/**
+ * Get transactions that match an account id in a set of transactions
+ * @param {number} accountId - The ID of the account
+ * @param {Object[]} transactions - The transactions to filter
+ * @returns {Object[]}
+ */
+export const getTransactionsForAccount = (accountId, transactions) =>
+  transactions.filter(t => t.account === accountId)
```

Comment:
```
transactions.filter(t => t.account === accountId)
getTransactionsForAccount(account._id, transactions)
```

Nit : the second line has more characters than the first and its meaning is less clear since it uses the name `get` when instead it is filtering. I find the first way easier to understand.